frontend/src/components/datasets/mainpage/SideCard.vue

```diff
@@ -6,31 +6,36 @@
 				<img v-else :src="dataset.thumbnail">
 			</div>
 			<h4>Details:</h4>
-			<p><strong>Year Created:</strong> {{dataset.year_created}}</p>
-			<p><strong>Size:</strong> {{dataset.size}}</p>
-			<p><strong>Number of Categories:</strong> {{dataset.num_cat}}</p>
-			<p><strong>COVE Profile Last Updated:</strong> {{dataset.date_created | moment}}</p>
+			<p v-if="dataset.year_created != null"><strong>Year Created:</strong> {{dataset.year_created}}</p>
```

Comment:
I think this was a nice visual update. Helps clear things up in the sidecard view.